naming_conventions.md

```diff
@@ -4,16 +4,16 @@ The page lists obligatory rules for our naming conventions. Please stick to it.
 
 ## 1 Project Name
 
-The project name <project-name> consists of <base-project>-<sub-project>, and only uses [a-z0-9] and dash -. No underline _.
+The project name [project-name] consists of [base-project]-[sub-project], and only uses [a-z0-9] and dash -. No underline _.
```

Comment:
yes, exactly my call (see also comment concerning the typo which @schmijos found).

the problem is that on gitbooks custom html "tags" ```<application>``` are invisible, since the browser doesn't know how to render them.

the next step would be to improve our cli, so that when we copy the template files we can replace the [project-name] placeholders with the real project name ;)